wsgi/odyssy/announcement/models.py

```diff
@@ -37,13 +37,20 @@ def get_single_announcement(announcement_id):
     def get_latest_announcements(cnt):
         """ get latest announcement """
         return Announcement.objects.filter(
-            Q(initDate__gte=datetime.datetime.today())|
-            Q(finDate__gte=datetime.datetime.today())
+            Q(initDate__gte=datetime.today())|
+            Q(finDate__gte=datetime.today())
             ).order_by('-initDate')[:cnt]
 
+    def get_default_start_date():
+        """ set default as start of date """
+        return datetime.combine(datetime.today(), time.min)
+
+    def get_default_end_date():
+        """ set end as undefined date """
+        return datetime.combine(datetime.max, time.max)
```

Comment:
The maximum date and the maximum time. From undefined i only get idea of this (9999-12-31 23:59:59)